packages/apollo-gateway/src/__tests__/integration/requires.test.ts

```diff
@@ -47,3 +51,362 @@ it('supports passing additional fields defined by a requires', async () => {
   expect(queryPlan).toCallService('product');
   expect(queryPlan).toCallService('books');
 });
+
+const serviceA: ServiceDefinitionModule = {
```

Comment:
I appreciate the rigor and depth of these test cases but they're admittedly a bit hard to reason about and validate (as a human). Is there any way we can apply the concepts here to our existing test fixtures? If that seems like a large ask (which would be understandable), can we humanize the types and field names in this example?

Reply:
I agree, they're not very human readable. I updated them with @lennyburdette's tests, since they're much nicer. Thanks Lenny!

Reply:
Glad we could collaborate! :grin: